src/Polysemy/Compose.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+module Polysemy.Compose
+  ( Compose (..)
+  , sendCompose
+  , runCompose
+  , subsumeCompose
+  , KnownList
+  ) where
+
+import Data.Proxy
+import Polysemy
+import Polysemy.Internal
+import Polysemy.Internal.Union
+
+data Compose :: EffectRow -> Effect where
```

Comment:
I'd prefer a name like `Bundle` or `Group` or something that more immediately indicates what's going on here.

Reply:
Sure.